[FEATURE] Améliorer l'accessibilité de l'infobulle sur le nombre de pix total

celineung opened the pull request:
## :unicorn: Problème
L'utilisateur au clavier n'a pas accès à l'icône "i" servant à afficher l'infobulle.

## :robot: Solution
- [X]  Rendre atteignable l'infobulle en l'encapsulant dans un button
- [X] Ajouter un attribut id à l’infobulle contenant la valeur de l'attribut aria-describedby du button servant à afficher l'infobulle
- [x]  Ajouter aria-label sur le button
- [x] Améliorer la lisiblité du texte dans l'infobulle (line-height, letter-spacing, word-spacing)
- [x] Sortir de l'infobulle si l'utilisateur appuie sur la touche Escape
- [x] Sortir de l'infobulle si le bouton perd le focus clavier

## :rainbow: Remarques
Documentation pour une infobulle accessible:
- https://www.w3.org/TR/wai-aria-practices/#tooltip
- https://www.accede-web.com/notices/interface-riche/infobulles-personnalisees/

## :100: Pour tester
> _Les instructions pour reproduire le problème, les profils de test, le parcours spécifique à utiliser, etc._

Commit:
feat: toggle score tooltip display from button

Commit:
test: add tests on tooltip display from hover on score

pix-service commented:
I'm deploying this PR to these urls:

- App: https://app-pr2985.review.pix.fr
- Orga: https://orga-pr2985.review.pix.fr
- Certif: https://certif-pr2985.review.pix.fr
- Admin: https://admin-pr2985.review.pix.fr
- API: https://api-pr2985.review.pix.fr/api/

 Please check it out!

Commit:
feat: add aria-label on pix-score button

Commit:
refacto: change tooltip display condition

Before this commit, when the tooltip is displayed, it takes a css class 'hexagon-score__information--visible'. When it is hidden, the css class is changed to 'hexagon-score__information--hidden'. Instead if a class change, this commit will display the tooltip on a variable change: if 'displayHelp' is true the tooltip is displayed, if 'displayHelp' is false the tooltip is hidden.

Commit:
feat: close tooltip on escape key pressed

alexandrecoin committed:
feat: close tooltip on focusout

Commit:
style: fix total pix score button display

Commit:
style: fix responsive tooltip width

Commit:
style: increase spacing in tooltip

alexandrecoin commented on mon-pix/tests/integration/components/hexagon-score_test.js:
```diff
@@ -35,4 +37,83 @@ describe('Integration | Component | hexagon-score', function() {
+        expect(find(tooltip)).not.to.exist;
+      });
+    });
+
```
**nitpick** : Retirer le saut de ligne.
